src/ModelIdentifier_TEST.cc

```diff
@@ -172,7 +172,7 @@ TEST(ModelIdentifier, AsString)
         "Name: \n"\
```

Comment:
Can this `else` be completely removed?

Reply:
this can not be remove it, there is a different between Windows and Linux, 

Linux can return a URL which ends on `/` and WIndows it doesn't . Both URL are valid.